app/brave_main_delegate.cc

```diff
@@ -164,6 +166,12 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   command_line.AppendSwitch(switches::kEnableDomDistiller);
   command_line.AppendSwitch(switches::kNoPings);
 
+  auto update_url = brave_component_updater::GetUpdateURLHost();
+  if (!update_url.empty()) {
```

Comment:
you need to ensure this is not empty for official builds

Reply:
also if we're overriding this using the command line switch, why do we need this? https://github.com/brave/brave-core/pull/9933/files#r703858783

Reply:
regardless it should never be empty for official builds

Reply:
I guess it's covered by GN, but wouldn't hurt to also check here